### ISSUE TITLE
Ambigious description of the Translation issue

### ISSUE THREAD

**Issue by harshit-maheshwari** [CyBear-Jinni/cbj_app#419]
Hi, I was checking translation file of Hindi(India)
having code hi-IN.json file and found out that there was no proper translation of the keywords.
I found a major issues with this translation file and this issue was Dominant and severe:

Blind is a homonyms and have different meanings in the english language and most prominent meaning of "blind" is "A person who can't see",
Similar is with word "scene".

This will totally be misguiding for the end-user as "Blinds" in the project is reffering to the "Curtain like covering of windows used for shade "

Unawareness of the context of the words can create blunders and can misguide end-user. 

There is a need to attach an extra file containing information such as:
1. Definitions of keywords
2. In which context it will be used
3. How to figure out which word is to be translated and which one is to be transliterated
4. This file must also contain a guideline stating that, "There should be no use of online translators", as these translators changes the type of verb (example: action verb to transitive verb)  

I found this error in the hi-IN.json file which was changing whole context of the word.  
I have shared a solution as a PR for that particular file but this issue can be seen in other languages too and it is needed to be corrected.

**Comment by guyluz11** [comment]
@harshit-maheshwari thanks for your help in the project it is very heartwarming.

I can see your concerts and I think it could help the project to have such a translation guiding file.

We are trying to make contributing and translations to the project as easy as it can be, so we are ok with wrong (hopefully not insulting) translations as we know that some users of the app will fix that when needed.
Advancing with time and always forward :).

A guiding file will surely help to reduce wrong translations but it will be much more work for developers that add a translation.
So we can use a file like that but it will "count" as another translation where it is not always updated to support the last translation existing (and that's ok).

_**In conclusion**_

A guiding file is needed, and pr for that is welcome.
